src/transfer.cpp

```diff
@@ -1889,31 +1896,31 @@ error TransferList::pause(Transfer *transfer, bool enable, DBTableTransactionCom
     return API_EFAILED;
 }
 
-transfer_list::iterator TransferList::begin(direction_t direction)
+auto TransferList::begin(direction_t direction) -> transfer_list::iterator
```

Comment:
Out of curiosity, what's the motivation of this change?

Reply:
transfer_list is now a type defined in TransferList (as a specialization of the lazy bulk erase deque), whereas before it was  global.   In order to name it in the old way, it would have to be TransferList::transfer_list::iterator which is getting a bit long and repetitive.  Using the new syntax with the type trailing after -> means it is evaluated in the class' context rather than global context, so we can leave off the TransferList::  which is one of the aims of that syntax enhancement in C++11 (besides also bringing it in line with lambda syntax).   thanks

Reply:
Thank you for the explanation, Matt.